lib/nxt_support/preprocessors/strip_preprocessor.rb

```diff
@@ -8,10 +9,12 @@ def initialize(value)
       end
 
       def call
-        return value unless value.is_a?(String)
+        return if value.nil?
 
         value.strip!
         value
+      rescue NoMethodError
+        raise WrongTypeError, 'Column type is not supported for this preprocessor'
```

Comment:
I think I would rather say here something like `Tried to call :strip! on #{value}. But value does not respond to :strip` as this says exactly what happend. The error message is quite cryptic if you think from user perspective.

Reply:
I doesn't even mention which column. And it might not even be a column